pkg/provider/constants.go

```diff
@@ -140,3 +141,70 @@ provider "kubernetes" {
   token                  = data.aws_eks_cluster_auth.cluster.token
 }
 `
+
+// TODO: Figure out how to deal with local backend
+// TODO: Figure out how to configure the Kubernetes provider (rke/equinix datasource or store somewhere?)
+const equinixBackendTemplate = `terraform {
+  backend "local" {
+    path = "../../{{ .Values.Bucket }}/{{ .Values.__CLUSTER__ }}/{{ .Values.Prefix }}/terraform.tfstate"
+  }
+{{- if .Values.ClusterCreated }}
```

Comment:
I feel like it might have been simpler to just configure the helm provider locally in the bootstrapping module.

Reply:
Probably haha. I was running into some issues with those providers before so that'll need some more attention and testing as well.